### ISSUE TITLE
segmenter infer error[Bug] 

### ISSUE THREAD

**Issue by dengguanguan** [open-mmlab/mmdeploy#2784]
### Checklist

- [X] I have searched related issues but cannot get the expected help.
- [X] 2. I have read the [FAQ documentation](https://github.com/open-mmlab/mmdeploy/tree/main/docs/en/faq.md) but cannot get the expected help.
- [ ] 3. The bug has not been fixed in the latest version.

### Describe the bug

/home/dengsj/SemanticSegmentation/mmlab/mmsegmentation/mmseg/models/losses/cross_entropy_loss.py:250: UserWarning: Default ``avg_non_ignore`` is False, if you would like to ignore the certain label and average loss over non-ignore labels, which is the same with PyTorch official cross_entropy, set ``avg_non_ignore=True``.
  warnings.warn(
Loads checkpoint by local backend from path: ./weights/segmenter_vit-s_mask_8x1_512x512_160k_ade20k_20220105_151706-511bb103.pth
The model and loaded state dict do not match exactly

unexpected key in source state_dict: backbone.patch_embed.projection.bias

### Reproduction

from mmseg.apis import inference_model, init_model, show_result_pyplot
import mmcv
import time

config_file = "configs/segmenter/segmenter_vit-s_mask_8xb1-160k_ade20k-512x512.py"
checkpoint_file = './weights/segmenter_vit-s_mask_8x1_512x512_160k_ade20k_20220105_151706-511bb103.pth'

# build the model from a config file and a checkpoint file
model = init_model(config_file, checkpoint_file, device='cuda:0')

# test a single image and show the results
img = 'demo/demo.png'  # or img = mmcv.imread(img), which will only load it once
for i in range(10):
    start_time = time.time()
    result = inference_model(model, img)
    end_time = time.time()
    print(f"模型推理时间为: {end_time-start_time}")

### Environment

```Shell
mmcv                   2.1.0
mmdeploy               1.3.1
mmdeploy-runtime-gpu   1.3.1
mmengine               0.10.4
mmsegmentation         1.2.2
```
